packages/forms/src/model/abstract_model.ts

```diff
@@ -166,19 +166,86 @@ export type ɵIsAny<T, Y, N> = 0 extends(1&T) ? Y : N;
 export type ɵTypedOrUntyped<T, Typed, Untyped> = ɵIsAny<T, Untyped, Typed>;
 
 /**
- * Value gives the type of `.value` in an `AbstractControl`.
+ * Value gives the value type corresponding to a control type.
  *
- * For internal use only.
+ * Note that the resulting type will follow the same rules as `.value` on your control, group, or
+ * array, including `undefined` for each group element which might be disabled.
+ *
+ * If you are trying to extract a useful value type, you probably want {@see RawValue}, which will
+ * not have `undefined` in group keys.
+ *
+ * @usageNotes
+ *
+ * ### `FormControl` value type
+ *
+ * You can extract the value type of a single control:
+ *
+ * ```ts
+ * type NameControl = FormControl<string>;
+ * type NameValue = Value<NameControl>;
+ * ```
+ *
+ * The resulting type is `string`.
+ *
+ * ### `FormGroup` value type
+ *
+ * Imagine you have an interface defining the controls in your group. You can extract the shape of
+ * the values as follows:
+ *
+ * ```ts
+ * interface PartyFormControls {
+ * address: FormControl<string>;
+ * }
```

Comment:
nit: do we have any auto-formatting when we extract docs? If not, we can apply some formatting here:
```suggestion
 * interface PartyFormControls {
 *   address: FormControl<string>;
 * }
```